state-chain/pallets/cf-elections/src/electoral_systems/mocks/access.rs

```diff
@@ -304,19 +308,22 @@ impl MockStorageAccess {
 		ELECTION_STATE.with(|old_state| {
 			let state_ref = old_state.borrow();
 			state_ref
-				.get(&identifier.encode())
+				.get(&identifier.unique_monotonic().encode())
 				.map(|v| ES::ElectionState::decode(&mut &v[..]).unwrap())
 				.unwrap()
 		})
 	}
 
 	pub fn set_election_properties<ES: ElectoralSystem>(
 		identifier: ElectionIdentifierOf<ES>,
-		properties: ES::ElectionProperties,
+		properties: Option<ES::ElectionProperties>,
 	) {
 		ELECTION_PROPERTIES.with(|old_properties| {
 			let mut properties_ref = old_properties.borrow_mut();
-			properties_ref.insert(identifier.encode(), properties.encode());
+			match properties {
+				Some(properties) => properties_ref.insert(identifier.encode(), properties.encode()),
+				None => properties_ref.remove(&identifier.encode()),
```

Comment:
maybe this is related to the question you asked this morning, but this shouldn't be necessary. should only need to remove properties as part of the election deletion

Reply:
I had to do this because on "refresh()", the a new identifier is created (with extra incremented by 1), which creates a new entry in the "election properties" map. In "force_consensus" we have to ensure there's only 1 property identifier, which will fail if we don't remove the previous identifier from the properties map. This logic is also consistent with the actual implementation of the "refresh_elections()" function